### DISCUSSION TITLE
How to create a badge with specific text?

### DISCUSSION THREAD

**Discussion by adamalston** [badges/shields#5536] [Q&A] [answered]
I am trying to create a badge that has the logo on the left and the text on the right with two different color backgrounds. Currently I have:

![](https://img.shields.io/badge/Tools-VSCode-informational?style=flat&logo=visual-studio-code&logoColor=007ACC&color=white)

`![](https://img.shields.io/badge/Tools-VSCode-informational?style=flat&logo=visual-studio-code&logoColor=007ACC&color=white)`

I am trying to have the logo on the left (without the word 'Tools') with a gray background, and the text on the right with a white background. How do I achieve this?

**Comment by chris48s** [comment, marked as answer]
Are either of these what you're looking for?
![](https://img.shields.io/badge/-VSCode-informational?style=flat&logo=visual-studio-code&logoColor=007ACC&color=white) https://img.shields.io/badge/-VSCode-informational?style=flat&logo=visual-studio-code&logoColor=007ACC&color=white
![](https://img.shields.io/badge/-VSCode-informational?style=flat&logo=visual-studio-code&logoColor=007ACC&color=white&labelColor=555) https://img.shields.io/badge/-VSCode-informational?style=flat&logo=visual-studio-code&logoColor=007ACC&color=white&labelColor=555

**Comment by adamalston** [comment]
Yes, thanks!
